go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/kubernetes-sigs/aws-fsx-csi-driver
 
 require (
-	github.com/aws/aws-sdk-go v1.16.5
+	github.com/aws/aws-sdk-go v1.16.36
```

Comment:
The reason I upgrade client SDK is because I meet the issue that ExportPath doesn't exist in v1.16.5.

Reply:
could you update the attribution as well: https://github.com/aws/csi-driver-amazon-fsx/blob/master/THIRD-PARTY#L1

Reply:
I see. Added in THIRD_PARTY